src/components/App/App.js

```diff
@@ -1,20 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Route, Switch } from 'react-router';
 import CurrentTour from '../CurrentTour/CurrentTour';
 import NavBar from '../NavBar/NavBar';
 import AddDate from '../AddDate/AddDate';
+import { userMockData } from '../../dataFiles/userData'
 import './App.css';
 
 const App = () => {
+  const [bookedShows, setBookedShows] = useState([])
+
+  useEffect(() => {
+    setBookedShows(userMockData)
+  }, [])
+
+  const updateShows = (newShow) => {
+    setBookedShows([...bookedShows, newShow])
+  }
+
   return (
     <main className="App">
       <NavBar />
       <Switch>
-        <Route exact path='/currentTour/' component={ CurrentTour } />
-        <Route exact path='/addDate/' component={ AddDate } />
+        {bookedShows &&
+        <Route exact path='/currentTour/' render={() => <CurrentTour userMockData={bookedShows}/>} />}
+        <Route exact path='/addDate/' render={() => <AddDate updateShows={updateShows}/>} />
```

Comment:
Clean Switch - Going to add an catch all as default?

Reply:
Absolutely! Thanks for the reminder